dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -40,6 +40,32 @@
 			status = "disabled";
 		};
 
+		adc1: adc@40022000 {
+			st,resolutions = <STM32_ADC_RES_16 STM32_ADC_RES_14
+					  STM32_ADC_RES_12 STM32_ADC_RES_10
+					  STM32H72x_ADC_RES_8>;
+		};
+
+		adc2: adc@40022100 {
+			st,resolutions = <STM32_ADC_RES_16 STM32_ADC_RES_14
+					  STM32_ADC_RES_12 STM32_ADC_RES_10
+					  STM32H72x_ADC_RES_8>;
+		};
+
+		adc1_2: adc@40022300 {
+			st,resolutions = <STM32_ADC_RES_16 STM32_ADC_RES_14
+					  STM32_ADC_RES_12 STM32_ADC_RES_10
+					  STM32H72x_ADC_RES_8>;
+		};
+
+		adc3: adc@58026000 {
+			st,resolutions = <STM32_ADC_RES_12 STM32_ADC_RES_10
+					  STM32H72x_ADC_RES_8
+					  STM32H72x_ADC3_RES_6>;
```

Comment:
Isn't the shift value wrong for the resolutions other than 6 bits?  It looks like ADC3 should use a shift of 3 for every resolution supported.

> 
> Bits 4:3 RES[1:0]: Data resolution
> These bits are written by software to select the resolution of the conversion.
> 00: 12-bit
> 01: 10-bit
> 10: 8-bit
> 11: 6-bit
>